src/current/cockroachcloud/folders.md

```diff
@@ -375,28 +235,16 @@ Only an empty folder can be deleted.
 To delete a folder:
 
 1. Move or delete all descendant folders and clusters.
-
-1. Delete the folder:
-
-    {% include_cached copy-clipboard.html %}
-    ~~~ shell
-    curl --request DELETE \
-        --url 'https://cockroachlabs.cloud/api/v1/folders/<folder-id>' \
-        --header 'Authorization: Bearer <api-key>' \
-        --header 'content-type: application/json'
-    ~~~
-
-    Replace:
-
-    - `<folder-id>`: The ID of the folder.
-    - `<api-key>`: Your service account's API key.
+1. Go to the **Clusters** page. The folders and clusters at the root of the organization are shown.
+1. Browse to the folder that contains the folder you want to delete.
+1. Next to the folder you want to delete, click the three-dots **Action** button and select **Delete folder**.
+1. Type the name of the folder to confirm, then click **Delete**.
 
 ## Limitations
 
 - Folders can be nested a maximum of four levels deep, including the organization level.
-- Each level of the hierarchy can contain a maximum of 200 direct descendant objects (folders or clusters).
-- Using folders requires the [CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}) or the [Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach) v1.1.0 or above. It is not yet possible to view the folder structure, move resources, or assign the `FOLDER_ADMIN` or `FOLDER_MOVER` roles from the {{ site.data.products.db }} Console or by using the `ccloud` command.
-- In the {{ site.data.products.cloud }} Console, all clusters that a user has permission to access are shown in a flat list. All other clusters are hidden.
+- An organization can have a maximum of 65 folders, regardless of how they are organized.
+- You can manage folders using the CockroachDB {{ site.data.products.cloud }} Console, the [CockroachDB {{ site.data.products.cloud }} API]({% link cockroachcloud/cloud-api.md %}), or the [Terraform provider](https://registry.terraform.io/providers/cockroachdb/cockroach) v1.1.0 or above. It is not yet possible to view the folder structure, move resources, or assign the `FOLDER_ADMIN` or `FOLDER_MOVER` roles by using the `ccloud` command.
```

Comment:
Maybe link to https://www.cockroachlabs.com/docs/cockroachcloud/ccloud-get-started

Reply:
I don't think I want to do that here since this feature specifically isn't supported by the `ccloud` command. WDYT?

Reply:
I thought of that, but landed on feeling that a link would provide helpful context either way. I leave it up to you, though.

Reply:
Yeah I think there are better places for them to discover the `ccloud` command than in this topic where the `ccloud` command won't even work to do what they need to do.